experimental/dds/tree2/src/test/feature-libraries/editable-tree/mockData.ts

```diff
@@ -154,14 +158,18 @@ export type Person = EditableTree &
 		"editable-tree.Test:Person-1.0.0"
 	>;
 
-export const personData: ContextuallyTypedNodeDataObject = {
+export const personData: SchemaAware.TypedField<
```

Comment:
for some reason, users of this which used it as initial tree content with schematize were able to compile despite it lacking strong types. This somehow broke, likley due to the changes to how toDocumentSchema was paramaterized.

This code has been updated to use more specific types as a fix, but its not clear exactly why this change was needed of how many other possible usages would be impacted.